src/constants/firebase.js

```diff
@@ -7,6 +7,10 @@ export type GetFirestoreType = Object
 
 export type IdType = string
 
+export type FirebaseAuthType = $ReadOnly<{
+  uid: IdType,
+}>
+
```

Comment:
I wrote this as ```//////////
// AUTH //
//////////

export type AuthType = $ReadOnly<{
  email: string,
  password: string,
}>```

can your definition `{uid: IdType = string}` be used in place of my definition when I merge?

Reply:
You'll have to add your fields there

Reply:
Right on, thanks!